src/styles/styles_for_all/all_elements.scss

```diff
@@ -0,0 +1,6 @@
+* {
```

Comment:
this file should be rather name:

`src/styles/main/main.scss`

it is more clear and descriptive name with good practice :)